packages/server/lib/hooks/connection/post-connection.ts

```diff
@@ -23,22 +22,12 @@ async function execute(createdConnection: RecentlyCreatedConnection, providerNam
     const { connection: upsertedConnection, environment, account } = createdConnection;
     let logCtx: LogContext | undefined;
     try {
-        const credentialResponse = await connectionService.getConnectionCredentials({
-            account,
-            environment,
-            connectionId: upsertedConnection.connection_id,
-            providerConfigKey: upsertedConnection.provider_config_key,
-            logContextGetter,
-            instantRefresh: false,
-            onRefreshSuccess: connectionRefreshSuccessHook,
-            onRefreshFailed: connectionRefreshFailedHook
-        });
-
-        if (credentialResponse.isErr()) {
+        const connectionRes = await connectionService.getConnection(upsertedConnection.connection_id, upsertedConnection.provider_config_key, environment.id);
+        if (connectionRes.error || !connectionRes.response) {
```

Comment:
On post connection I don't think we need to test again the credentials since we just fetched them, but maybe I forgot an edge case? @khaliqgant

Reply:
No, should be fine to just use `getConnection` here